classes/image_types_ostree.bbclass

```diff
@@ -24,10 +24,17 @@ TAR_IMAGE_ROOTFS:task-image-ostree = "${OSTREE_ROOTFS}"
 
 OSTREE_RMDIR_HELPER_MSGTYPE ?= "bbwarn"
 ostree_rmdir_helper(){
-    if ! rmdir ${1}; then
-        ${OSTREE_RMDIR_HELPER_MSGTYPE} "Data in ${1} directory is not preserved by OSTree. Consider moving it under /usr\n$(find ${1})"
-        rm -vrf ${1}
+    if [ -d ${1} ]; then
+        rmdir ${1} && return 0
```

Comment:
This line seems redundant and will skip the subsequent message.

Reply:
The subsequent message will only be skipped when the directory is empty because otherwise `rmdir` will fail.
Maybe we should say something in these cases too (when we delete only the directory)?

Reply:
Oh, I get it, sorry.

> Maybe we should say something in these cases too (when we delete only the directory)?

If it was silent before, it's fine to keep it silent, otherwise, sure, a brief message is not a bad idea.

Reply:
to keep it simple I have changed the way of cleaning up `/var` and maintain the single message.
updated the pr description too.